### PR TITLE
typo fixes in vars and plaintext

### PR DESCRIPTION
**Pull request by davidpesce**
Fixes for #894 along with a few other typos in vars and plaintext.

### CODE REVIEW

File: app/lang/en/users.php [hunk 1]
```diff
@@ -52,6 +52,6 @@
   'updated_error'  => 'There was a problem updating that account.',
   'registration' => array(
       'thanks'       => 'Thanks for signing up to use Learning Locker. To complete your registration, we need you to verify your email.',
-      'instructions' => 'Please go to the following web address:'
```
Comment (ryansmith94) [inline app/lang/en/users.php]: Ah sorry @davidpesce, I thought you'd intended to change the code that uses this. I'm actually not sure whether it's better to change it here or there. If you're certain that "instructions" hasn't been used anywhere else then this is fine.

**Comment by ryansmith94** [comment]
The build is failing because of #896. In the meantime @davidpesce, you can merge the "issue/897" branch into your branch to fix the problem.

**Comment by davidpesce** [comment]
OK, I'll merge #896 and look into the code that uses it.

**Comment by ryansmith94** [comment]
Awesome thanks @davidpesce

**Comment by davidpesce** [comment]
users.registration.instructions wasn't used anywhere. users.registration.click was a correct fix.

the lrs.client.created_sucecss was misspelled in one place, but correct in the others. I fixed the misspelling in app/controllers/ClientController.php.

Should be good to go!

**Comment by ryansmith94** [comment]
Ok nice one.

Ah good spot, thanks for fixing that 😄

**Pull request merged**